executor/show.go

```diff
@@ -175,6 +175,14 @@ func (e *ShowExec) fetchShowProcessList() error {
 		if len(pi.Info) != 0 {
 			t = uint64(time.Since(pi.Time) / time.Second)
 		}
+
+		var info string
+		if e.Full {
```

Comment:
Could you add a test case?

Reply:
I tried adding test but failed.

Since testSuite in tidb/executor  has no SessionManager,  full tests on show processlist seems not possible here.  

Can you give me some advice?

Reply:
You can create a util/mock.Context and set a mocked SessionManager to it.